pallets/subtensor/src/utils/misc.rs

```diff
@@ -96,6 +96,58 @@ impl<T: Config> Pallet<T> {
         Self::deposit_event(Event::OwnerHyperparamRateLimitSet(epochs));
     }
 
+    pub fn cancel_deregistration_priority_schedule_for_owner(owner: &T::AccountId) {
```

Comment:
```suggestion
    pub fn cancel_subnet_deregistration_priority_schedules_for_owner(owner: &T::AccountId) {
```
See prev comment + It's important to keep "scheduleS" in the name as it clears all schedules and not only a single one